packages/message/src/PDRs.ts

```diff
@@ -0,0 +1,73 @@
+import { Message } from '@cumulus/types';
+
+interface PDR {
+  name: string
+  PANSent: boolean
+  PANmessage: string
+}
+
+interface MessageWithPdr extends Message.CumulusMessage {
+  payload: {
+    pdr?: PDR
```

Comment:
It isn't entirely clear to me when we want to actually update our core `CumulusMessage` type. I think there was some reason that we intentionally kept `payload` as unknown for that type, because it can be so different in different cases. So it seemed appropriate to create a custom type for this case.

Reply:
OK re: sticking with an override for this case.

How about`MessageWithOptionalPayloadPdr` for the name?

Reply:
Sure, that seems fine.

Reply:
bdce8ab59a5a77fec0e36b405ac0caaee97a7120